packages/compodoc/src/executors/build/schema.d.ts

```diff
@@ -56,6 +56,13 @@ export interface CompodocOptions {
   /** @default false */
   disableDependencies: boolean;
 
+  /** @default 0 when coverage is deactivated, 70 otherwise */
+  coverageTest?: number;
```

Comment:
Is there a reason to mark this as optional?
```suggestion
  coverageTest: number;
```